synapse/app/homeserver.py

```diff
@@ -97,9 +97,24 @@ def build_resource_for_federation(self):
         return JsonResource(self)
 
     def build_resource_for_web_client(self):
-        import syweb
-        syweb_path = os.path.dirname(syweb.__file__)
-        webclient_path = os.path.join(syweb_path, "webclient")
+        webclient_path = self.get_config().web_client_location
+        if not webclient_path:
+            try:
+                import syweb
+            except ImportError:
+                quit_with_error(
+                    "Could not find a webclient.\n\n"
+                    "Please either install the matrix-angular-sdk or configure\n"
+                    "the location of the source to serve via the configuration\n"
+                    "option `web_client_location`\n\n"
+                    "To install the `matrix-angular-sdk` via pip, run:\n\n"
+                    "    pip install 'matrix-angular-sdk'\n"
```

Comment:
^ Does this actually work.
Doesn't it have to be https://github.com/matrix-org/matrix-angular-sdk/tarball/v0.6.6/ or something?